### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by nightlight-wander**
avatar-badge component ,although it has been already reviewed ,i only added code section .So just wanted a review for that, if it looks okay.
color of one file is not being reflected on hosting, otherwise while using live server it works, may be having issue with highlight.js library,i will look into it later.
I have attached a snapshot of actual [color.]
avatar code
[https://hungry-heisenberg-c55f9e.netlify.app/](url)

### CODE REVIEW

File: avatar-badge/avatar-badge.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    width: 1.3rem;
+    bottom: 1.3rem;
+    right: 0.4rem;
+    border-radius:50% ;
+    border: 2px solid var(--blue-purple);
+
+}
+
+.online{
+    background-color:#00edc6;
```
Comment (yashpatel1797) [inline avatar-badge/avatar-badge.css:57]: As you have used variables previously you can use them here too.

File: avatar-badge/avatar-badge.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.offline{
+    background-color: #8089c7;
```
Comment (yashpatel1797) [inline avatar-badge/avatar-badge.css:61]: same as above.

File: avatar-badge/avatar-badge.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// const codeElems=document.querySelectorAll(".code")
+// codeElems.forEach( element=>{
+//      element.classList.remove('.language-xml').add('.language-html')
+//      console.log(element)
+
+// })
```
Comment (yashpatel1797) [inline avatar-badge/avatar-badge.js:6]: can remove comments if not useful.

**Review by yashpatel1797 — COMMENTED** [review]
The code looks good. just a small suggestion is that for background color you can use lighter shades of blue this is too dark. not mandatory just a suggestion.

File: avatar-badge/avatar-badge.js (hunk 3 above)
Comment (nightlight-wander) [reply avatar-badge/avatar-badge.js:6]: yeah i will do that,thanks!

File: avatar-badge/avatar-badge.css (hunk 2 above)
Comment (nightlight-wander) [reply avatar-badge/avatar-badge.css:61]: okeiy, i forgot it ,will do

**Comment by nightlight-wander** [comment]
okeiy , i will try to change color in last ,for now it was creating confusion, thank you! :)

**Pull request merged**